src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static String getRandomColor() {
+
+        return String.valueOf(Color.values()[new Random().nextInt(Color.values().length)]);
```

Comment:
`Random` must be a private field
```suggestion
        return String.valueOf(Color.values()[random().nextInt(Color.values().length)]);
```